src/System.Memory/src/System.Memory.csproj

```diff
@@ -9,6 +9,7 @@
     <DefineConstants Condition="'$(IsPartialFacadeAssembly)' != 'true'">$(DefineConstants);netstandard</DefineConstants>
     <DefineConstants Condition="'$(TargetGroup)'=='netcoreapp'">$(DefineConstants);netcoreapp</DefineConstants>
     <DefineConstants Condition="'$(TargetGroup)'=='netstandard1.1'">$(DefineConstants);netstandard11</DefineConstants>
+    <DefineConstants>$(DefineConstants);FEATURE_PORTABLE_SPAN</DefineConstants>
```

Comment:
I think there needs to be a condition for this constant (either IsPartialFacadeAssembly != true, or use the existing netstandard).

Reply:
I thought about that but found we include only the needed files when building the portable library. I can add the condition but I am not seeing this is necessary. I'll add the condition anyway.